src/DonationForms/resources/registrars/templates/fields/Amount/DonationAmountLevels.tsx

```diff
@@ -0,0 +1,45 @@
+import classNames from 'classnames';
+
+type DonationAmountLevelsProps = {
+    name: string;
+    currency: string;
+    levels: number[];
+    onLevelClick?: (amount: number) => void;
+};
+
+/**
+ * @since 0.2.0
```

Comment:
As this was moved from another file, I think we also need an `unreleased` tag here about this change.

Reply:
Resolved 5c6db24b